src/Orchard/Environment/ShellBuilders/ShellContextFactory.cs

```diff
@@ -103,7 +103,8 @@ public class ShellContextFactory : IShellContextFactory {
                 Features = new[] {
                     new ShellFeature { Name = "Orchard.Setup" },
                     new ShellFeature { Name = "Shapes" },
-                    new ShellFeature { Name = "Orchard.jQuery" },
+                    new ShellFeature { Name = "Orchard.Recipes" },
+                    new ShellFeature { Name = "Orchard.jQuery" }

```

Comment:
It's not actually, good catch. It's a leftover of some of the refactoring I did but forgot to undo this. I'll get rid of it.